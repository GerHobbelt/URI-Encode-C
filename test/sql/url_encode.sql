CREATE EXTENSION url_encode;

SELECT url_encode('Hello World');
SELECT url_decode('Hello%20World');

SELECT url_encode(unnest) from unnest(string_to_array('http://hu.wikipedia.org/wiki/SÃ£o_Paulo','/'));
