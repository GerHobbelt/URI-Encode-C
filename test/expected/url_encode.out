CREATE EXTENSION url_encode;
SELECT url_encode('Hello World');
  url_encode   
---------------
 Hello%20World
(1 row)

SELECT url_decode('Hello%20World');
 url_decode  
-------------
 Hello World
(1 row)

SELECT url_encode(unnest) from unnest(string_to_array('http://hu.wikipedia.org/wiki/SÃ£o_Paulo','/'));
    url_encode    
------------------
 http%3A
 
 hu.wikipedia.org
 wiki
 S%C3%A3o_Paulo
(5 rows)

